wordlist.py

```diff
@@ -1,14 +1,15 @@
 #!bin/python
 # -*- coding: UTF-8 -*-
+import config
+
 
 class MissingTranslationException(Exception):
     pass
 
+
 translations = {
     "en_US": [
-        [
-            'The space',
-        ],
+        config.SPACE_NAME,
```

Comment:
Maybe it would be easier to just have this be a list with some generic name("Hackspace", etc.) and a special placeholder string that is optional and only used when this variable is actually set in the config? Otherwise without the variable this would raise an exception when not set or set to something wrong.

Reply:
I am not sure of what you want then... if we leave the lists be and add a special name, this one will necessarily have to be a string else it will cause some issues. Is that what you want? For example, set the SPACE_NAME to a random default string such as "Hackspace" and then, let the rest of translations stay in the original list?

Reply:
Maybe leaving the list like it is but replacing the `Bytespeicher` item with one having a format string placeholder like `%{SPACE_NAME}` and parse the SPACE_NAME variable into that?